cmd/config/internal/commands/cmdcreatesetter.go

```diff
@@ -124,6 +126,16 @@ func (r *CreateSetterRunner) preRunE(c *cobra.Command, args []string) error {
 		r.CreateSetter.Description = r.Set.SetPartialField.Description
 		r.CreateSetter.SetBy = r.Set.SetPartialField.SetBy
 		r.CreateSetter.Type = r.Set.SetPartialField.Type
+
+		if r.CreateSetter.Type == "array" {
+			if !c.Flag("field").Changed {
+				return errors.Errorf("field flag must be set for array type setters")
+			}
+
+			// set it to empty string as the type is array and we derive the values at later point
+			// from filed path
+			r.CreateSetter.FieldValue = ""
```

Comment:
This shouldn't be needed since empty string is the nil value for `FieldValue`.

Reply:
The aim is to set it to empty even it user specifies some value as input.